packages/core/src/editor/Block.css

```diff
@@ -127,6 +127,12 @@ NESTED BLOCKS
 [data-content-type="heading"][data-level="3"] {
   --level: 1.3em;
 }
+[data-content-type="heading"][data-level="4"] {
+  --level: 1em;
+}
+[data-content-type="heading"][data-level="5"] {
+  --level: 0.9em;
+}
```

Comment:
Used https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/Heading_Elements
as reference for appropriate sizes